nexus/db-queries/src/db/datastore/deployment.rs

```diff
@@ -1202,7 +1202,7 @@ mod tests {
             );
             for (sled_id, agent) in &collection.sled_agents {
                 // TODO-cleanup use `TypedUuid` everywhere
-                let sled_id = TypedUuid::from_untyped_uuid(*sled_id);
+                let sled_id = SledUuid::from_untyped_uuid(*sled_id);
```

Comment:
Particularly like that this is now explicit in a way that's still convenient, and much nicer than `TypedUuid::<SledKind>::from_untyped_uuid`.